packages/store/src/translations/de.ux-store.json

```diff
@@ -0,0 +1,5 @@
+{
+    "error": {
+        "unsupportedEntity": "Nicht unterstützte Entität - {{entityName}}"
+    }
+}
```

Comment:
Should we merge partial translation?

Reply:
Do you mean there's only one entry in the German translation? I thought i18next falls back to the default language if the translation is not found. That file is there as a place holder and a reminder that we need to support more than US-EN eventually.

Reply:
@devinea, not sure I understood your question. Could you clarify please?

Reply:
@ullasholla I suppose my concern was if there is 1 text in German that the use might see it and other texts in English. Better all texts in the same language when shown to the users right?

Reply:
Ah, I see. Like I said, that's a placeholder to help us when we fully support translations. I think i18next falls back to just showing the keys, I'm reluctant to manually merge the values and would like to defer to i18next's implementation. I'll mark this as resolved for the moment. We need to put in a lot of work to get translations working anyway.